lua/catppuccin/init.lua

```diff
@@ -129,7 +129,7 @@ function M.setup(user_conf)
 	M.flavour = get_flavour(M.options.flavour or vim.g.catppuccin_flavour)
 
 	-- Caching configuration
-	local cached_path = M.options.compile_path .. M.path_sep .. "date.txt"
+	local cached_path = M.options.compile_path .. M.path_sep .. "date"
```

Comment:
Since the folder is changing this would be a good time to change the name to something more descriptive like, `checksum`, `cached`, etc.